### PR TITLE
Add new exercise custom-set

### PR DESCRIPTION
**Pull request by matthewmorgan**
As described in #154

### CODE REVIEW

File: custom-set/custom-set.spec.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import CustomSet from './custom-set';
+
+describe('CustomSet', () => {
+
+  xit('can delete elements', () =>{
```
Comment (rchavarria) [inline custom-set/custom-set.spec.js]: Whoops, we should enable at least the first test, right?

**Comment by rchavarria** [comment]
It looks like `example.js` contains only ES5 code, without any ES2015 feature. Are you ok with it? Wouldn't it be better to refactor it to some kind of ES2015 code?

**Comment by matthewmorgan** [comment]
I'm sorry-- I must have committed the wrong file.  I'll resubmit, and also enable the first test.

**Comment by matthewmorgan** [comment]
@rchavarria all set on the changes.  Please have a look.

**Comment by rchavarria** [comment]
Awesome! Another new exercise :tada:

**Pull request merged**